File: 준영/문자열 내 p와 y의 개수.py
def solution(s):
    s = s.lower()
    p_cnt = 0
    y_cnt = 0
    for i in s:
        if i == 'p':
            p_cnt += 1
        if i == 'y':
            y_cnt += 1

    if p_cnt == y_cnt:
        answer = True
    else:
        answer = False

    return answer

File: 준영/프로그래머스/LV1/문자열 내 p와 y의 개수.js
function solution(s){
  var answer = true;
  s = s.toLowerCase()
  var p_cnt = 0
  var y_cnt = 0
  for (const i of s) {
    if (i === 'p') {
      p_cnt += 1
    } 
    if (i === 'y') {
      y_cnt += 1
    } 
  }
  if (p_cnt === y_cnt) {
    answer = true
  } else {
    answer = false
  }
  return answer;
}